site/docs/windows.md

```diff
@@ -177,6 +181,12 @@ To build C++ targets, you need:
     set BAZEL_VC=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC
     ```
 
+    and to specify the exact Visual C++ Build Tools version:
+
+    ```
+    set BAZEL_VC_FULL_VERSION=14.16.27023
```

Comment:
Would it make more sense to include this in BAZEL_VC?
Introducing a new envvar seems to have no benefit.

Reply:
We cannot do that without changing the definition of BAZEL_VC, which would be an incompatible change.
Normally, users should have one version of VC build tools installed so they don't need this. This is only useful when users have multiple version installed and they want to use an old version.